docker/graphdb/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ontotext/graphdb:10.5.1
+RUN mkdir -p /opt/graphdb/dist/data/repositories/langchain
+COPY config.ttl /opt/graphdb/dist/data/repositories/langchain/
+COPY starwars-data.trig /
```

Comment:
Build will fail right now because some of these data doesn't exist locally.

We could add a script to download and verify hash or else avoid loading in data for now. For the integration tests,  it's useful to have the service even without fixtures loaded

Reply:
Yeah, correct. I have removed them.